modules/oe_content_event/config/install/field.field.node.oe_event.field_description_title.yml

```diff
@@ -0,0 +1,18 @@
+langcode: en
+status: true
+dependencies:
+  config:
+    - field.storage.node.field_description_title
+    - node.type.oe_event
+id: node.oe_event.field_description_title
+field_name: field_description_title
+entity_type: node
+bundle: oe_event
+label: 'Description title'
+description: ''
+required: false
```

Comment:
Should be required.

Reply:
Done through constraint. By documentation this field is required only if any fields within Description fields group filled in.